README.md

```diff
@@ -62,6 +62,21 @@ Check paddle job status
 $ kubectl -n paddle-system get pdj
 ```
 
+### Work with Volcano
+
+Enable volcano before installation, add the following args in *deploy/v1/operator.yaml*
+```
+containers:
+- args:
+  - --leader-elect
+  - --namespace=paddle-system # watch this ns only
+  - --scheduling=volcano      # enable volcano
```

Comment:
l think it’s better to add scheduling config at job.spec level instead of controller level, what do you think?

Reply:
Actually, global scheduling config must exist since pg crd may not exist in system; for job level config, I left it to pod spec without adding another field, just like tf-operator did.